examples/websocket/server/websocket.go

```diff
@@ -27,7 +27,7 @@ type wsCodec struct {
 
 func (wss *wsServer) OnBoot(eng gnet.Engine) gnet.Action {
 	wss.eng = eng
-	log.Printf("echo server with multi-core=%t is listening on %s\n", wss.multicore, wss.addr)
+	logging.Infof("\"echo server with multi-core=%t is listening on %s", wss.multicore, wss.addr)
```

Comment:
I believe the `"\` should be removed.

Reply:
I'm sorry, maybe you were looking at the code submission earlier, but this issue has been dealt with.

Reply:
<img width="775" alt="image" src="https://user-images.githubusercontent.com/13690966/153594631-d7744f9d-5751-47c2-aa97-4637eab96ffb.png">